spk/urbackup/Makefile

```diff
@@ -0,0 +1,59 @@
+SPK_NAME = urbackup
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/urbackup.png
+DSM_UI_DIR = app
+
+DEPENDS  = cross/busybox cross/$(SPK_NAME) 
+
+MAINTAINER = Josef
+DESCRIPTION = UrBackup is an easy to setup Open Source client/server backup system, that through a combination of image and file backups accomplishes both data safety and a fast restoration time.
+ADMIN_URL = /urbackup/
+RELOAD_UI = yes
+DISPLAY_NAME = Urbackup
+CHANGELOG = " urbackup"
+BETA = 1
+PARALLEL_MAKE=4
```

Comment:
Please remove. Not expected here. Use command line options.

Reply:
changed